WordPress/gutenberg
# Remove the gutenberg-block-directory experimental flag

This removes the flag and its UI, enabling the block directory by default.

Note that it does not change the API namespace, which is still `__experimental`.

See #23384.

## How has this been tested?
Manually tested, by verifying that the block directory is searched from the editor:
1. Visit Posts / Add New Post
2. Click Add Block
3. In Search For A Block, type`boxer`, `waves`, or `bliss`.
4. Confirm that some blocks from the directory are shown, headed with the text "No blocks found in your library. These blocks can be downloaded and installed:"

## Types of changes
- Removes conditional checks for the `gutenberg-block-directory` experiment.
- Removes the settings field.
- Removes the related setup from phpunit bootstrap.
- In short, enables the Block Directory by default.

## Checklist:
